### ISSUE TITLE
IATI: converting to spreadsheet doesn't generate a Meta sheet

### ISSUE THREAD

**Issue by stevieflow** [OpenDataServices/cove#1262]
I can understand why this is an expected behaviour - but this means it not possible to fully roundtrip, without adding in the Meta sheet (specially with the version and generated-datetime attributes)

Perhaps another way to express this is: the iati-activities element and attributes are lost on converting to a spreadsheet

**Comment by Bjwebb** [comment]
I agree. Its not a quick fix though, so we need to talk about scheduling some time for this.

**Comment by stevieflow** [comment]
@Bjwebb for the trello board?

**Comment by Bjwebb** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [IATI: converting to spreadsheet doesn't generate a Meta sheet](https://trello.com/c/Hs6N0nI5/35-iati-converting-to-spreadsheet-doesnt-generate-a-meta-sheet)

**Comment by Bjwebb** [comment]
flatten-tool issue for this https://github.com/OpenDataServices/flatten-tool/issues/176
